python2/httplib2/__init__.py

```diff
@@ -2133,13 +2135,19 @@ def request(
                         cachekey,
                     )
         except Exception as e:
+            isTimeout = isinstance(e, socket.timeout)
```

Comment:
Please change it to uniform snake case in both py2/3 and I'll merge it.

```suggestion
            is_timeout = isinstance(e, socket.timeout)
```